omx/gstomxaacdec.c

```diff
@@ -79,7 +81,7 @@ gst_omx_aac_dec_class_init (GstOMXAACDecClass * klass)
 static void
 gst_omx_aac_dec_init (GstOMXAACDec * self)
 {
-  /* FIXME: Other values exist too! */
+  /* NOTE: If necessary adjust to match TI aacdec setup */
   self->spf = 1024;
```

Comment:
We could create a definition for this number, it's similar to the GST_OMX_AAC_ENC_INBUF_NSAMPLES for the AAC encoder case.

Reply:
66cee90a7d9904dfc8155591da324492d4f520ef